library/Backends/Plonk/src/Juvix/Backends/Plonk/Parameterization.hs

```diff
@@ -154,16 +155,15 @@ arityRaw = \case
   PLte -> 2
   PEq -> 2
 
-toArg :: App.Return' ext1 ty term -> Maybe (App.Arg' ext2 ty term)
-toArg App.Cont {} = Nothing
-toArg App.Return {retType, retTerm} =
-  Just $
-    App.TermArg $
-      App.Take
-        { usage = Usage.Omega,
-          type' = retType,
-          term = retTerm
-        }
+toArg :: App.Return' ext1 ty term -> Maybe (App.Arg' ext1 ty term)
+toArg = Just . App.TermArg
+
```

Comment:
I'm guessing we can do this more minimal version as we keeping the same extension rather than changing it?